.env

```diff
@@ -54,6 +54,9 @@ DRUPAL_STAGE_FILE_PROXY_ORIGIN=https://www.your-site-domain.example/
 # Shield message.
 DRUPAL_SHIELD_PRINT="Restricted access."
 
+# Disable Drupal Shield for all environments (0 = enabled, 1 = disabled).
+DRUPAL_SHIELD_DISABLED=0
```

Comment:
please remove this - it is supposed to be planted only in the hosting environments